ecommerce/OrderController.php

```diff
@@ -283,10 +283,43 @@ public function make_refund() {
 			$this->json_response( tutor_utils()->error_message( HttpHelper::STATUS_UNAUTHORIZED ), null, HttpHelper::STATUS_UNAUTHORIZED );
 		}
 
+		$order_id = Input::post( 'order_id' );
```

Comment:
We need to follow WPCS

Reply:
Please provide some suggestion for this.